docs/source/deployment_guide_overview.md

```diff
@@ -0,0 +1,151 @@
+
+* Using secret objects to securely store important configuration files in your cluster. For information about Kubernetes secrets, check out [Kubernetes secrets](https://kubernetes.io/docs/concepts/configuration/secret/).
+* Cluster considerations and node sizing. In step 2 above, we discussed a general outline for how to think about the sizings of nodes. Your use case, as well as a robust period of development, is the only way you will truly know how how large your peers, ordering nodes, and CAs will need to be.
+* How you choose to mount your volumes. It is a best practice to mount the volumes relevant to your nodes external to the place where your nodes are deployed. This will allow you to reference these volumes later on (for example, to deploy nodes with virtually identical configurations) without having to enter the container where the node has been deployed.
+* How you will monitor your resources. It is critical that you establish a strategy and method for monitoring the resources used by your individual nodes and the resources deployed to your cluster generally. As you join your peers to more channels, you will need likely need to increase its CPU and memory allocation. Similarly, you will need to make sure you have enough storage space for your state database and blockchain.
+
+## Step three: Set up your CAs
+
+The first component that must be deployed in a Fabric is a CA. This is because the configuration of a component (such as a peer, an ordering node, or the MSP representing an organization) must include at least one user identity that is authorized to operate it before it can be deployed.
+
+**In a production network, it is recommended to deploy two CAs per organization to deploy all of the certificates associated with an organization.** For example, if you deploy three peers that are associated with one organization and an ordering node that is associated with another organization, you will need four CAs. Two of the CAs will be for the peer organization (generating the certificates for the peer, admins, communications, and the folder structure of the MSP representing the organization) and the other two will be for the orderer organization (generating the certificates for the ordering node, admins, and the folder structure of the ordering organization).
+
+These two CAs have a related role in generating the certificates associated with the organization.
+
+* One CA is used to generate (through a process called "enrollment") the certificates of the admin of an organization, the MSP of that organization, and any nodes owned by that organization. This CA will also generate the certificates for any additional users. Because of its role in "enrolling" identities, this CA is sometimes called the "enrollment CA" or the "ecert CA".
```

Comment:
One of the things that is missing here is whether to use a Fabric CA at all. Should I bring my own CA? should I use a separate Fabric CA for my blockchain needs? Should I bootstrap the Fabric CA with a certificate and key that is from a separate CA. I would be interested in a discussion between the trade-offs and recommendations.